pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -144,7 +216,7 @@ export default (
       if (this.type !== 'cron' || !this.value) {
         return;
       }
-      if (!isValidCron(this.value)) {
+      if (typeof this.value === 'string' && !isValidCron(this.value)) {
```

Comment:
This seems like it should be pushed into isValidCron. The function should enforce it's own type expectations.

Reply:
I would agree, but `isValidCron` is provided by the `cron-validator` package.